modules/sor/sorV2/sorV2.service.ts

```diff
@@ -0,0 +1,243 @@
+import { 
+    BasePool, 
+    ChainId,  
```

Comment:
can we get fantom added as a chainId in the SDK as well, please :)

Reply:
Think it should be already